### PR TITLE
Add Alex Russell-Saw to team page

### PR DESCRIPTION
**Pull request by arussellsaw**
🙌 First PR!

### CODE REVIEW

**Comment by Strum355** [comment]
If you want you can add yourself to https://github.com/sourcegraph/about/blob/main/company/team/locations.geojson too : )

**Comment by arussellsaw** [comment]
ah will do!

File: company/team/locations.geojson [hunk 1]
```diff
@@ -352,6 +352,19 @@
...
+    {
+      "type": "Feature",
+      "properties": {
+        "name": "Alex Russell-Saw"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+					51.5085300,
+          -0.1257400
```
Comment (Strum355) [inline company/team/locations.geojson:364]: I think these two need to be swapped, assuming UK. iirc I made the same mistake :sweat_smile: 
```suggestion
			-0.1257400,
			51.5085300
```

File: company/team/locations.geojson [hunk 2]
```diff
@@ -365,5 +365,18 @@
...
+    {
+      "type": "Feature",
+      "properties": {
+        "name": "Alex Russell-Saw"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          -0.1257400,
+					51.5085300
```
Comment (arussellsaw) [inline company/team/locations.geojson:377]: ugh bad indentation, an inauspicious first PR. will fix after a call

**Review by tsenart — APPROVED** [review]

**Pull request merged**